chart/templates/mirror-public-certificate.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if eq .Values.mirrorServiceTLS true }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ include "chart.fullname" . }}-public-cert
+spec:
+  dnsNames:
+    - full-service-mirror-public.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
Might be nice to add to the SANs list so we can use short names? 
 
- full-service-mirror-public
- full-service-mirror-public.{{ .Release.Namespace }}

Reply:
Yes totally